app/src/main/res/layout/review_fragment.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <TextView
+      android:id="@+id/dummy_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="This is dummy TextView for testing"
```

Comment:
To fully resolve #142 don't we also need to actually display the concept card from #160?

Reply:
Removed this file as it was already covered in `ConceptCardFragment`